vmdb/app/models/mixins/authentication_mixin.rb

```diff
@@ -46,8 +46,12 @@ def authentication_password_encrypted(type = nil)
     authentication_component(type, :password_encrypted)
   end
 
+  def required_credential_fields(_type)
```

Comment:
Sorry @himdel, still seeing more things...

I don't understand why this method is taking an unused type here and in ems_openstack_mixin.rb?  I don't see `required_credential_fields` defined elsewhere.  Can't we skip passing the `_type` in both cases?  Am I misunderstanding?